src/react-components/ory/user-auth-card.tsx

```diff
@@ -113,6 +115,8 @@ export const UserAuthCard = ({
       hasWebauthn(flow.ui.nodes) ||
       hasLookupSecret(flow.ui.nodes))
 
+  const showIdentifier = hasHiddenIdentifier(flow.ui.nodes)
```

Comment:
Would be good to have a comment explaining why `hasHiddenIndentifier` leads to showing the identifier. I was confused for a second.